agent/config/beerocks_agent.conf.in

```diff
@@ -29,20 +29,17 @@ backhaul_wire_iface=@BEEROCKS_BH_WIRE_IFACE@
 [agent0]
 radio_identifier=00:00:00:00:00:00
 hostap_iface_type=WIFI_INTEL
-hostap_iface=@BEEROCKS_WLAN1_IFACE@
```

Comment:
Also as part of this commit - can't we do the CMAKE changes as well??

Reply:
the `CMAKE` changes will require `prplmesh_utils.sh.in` changes and also `beerocks_controller.conf.in` changes.

we can create a task for it, but leave it out of this PR for reviewability

Reply:
OK - please add a task for such cleanups.

Reply:
the following issue will handle the above-mentioned required changes:
https://github.com/prplfoundation/prplMesh/issues/795